components/telemetry-operator/controller/tracepipeline/controller.go

```diff
@@ -178,23 +177,12 @@ func (r *Reconciler) doReconcile(ctx context.Context, pipeline *telemetryv1alpha
 		return fmt.Errorf("failed to create otel collector open census service: %w", err)
 	}
 
-	if r.config.CreateServiceMonitor {
-		serviceMonitor := makeServiceMonitor(r.config)
-		if err = controllerutil.SetControllerReference(pipeline, serviceMonitor, r.Scheme); err != nil {
-			return err
-		}
-
-		if err = utils.CreateOrUpdateServiceMonitor(ctx, r.Client, serviceMonitor); err != nil {
-			return fmt.Errorf("failed to create otel collector prometheus service monitor: %w", err)
-		}
-
-		metricsService := makeMetricsService(r.config)
-		if err = controllerutil.SetControllerReference(pipeline, metricsService, r.Scheme); err != nil {
-			return err
-		}
-		if err = utils.CreateOrUpdateService(ctx, r.Client, metricsService); err != nil {
-			return fmt.Errorf("failed to create otel collector metrics service: %w", err)
-		}
+	metricsService := makeMetricsService(r.config)
+	if err = controllerutil.SetControllerReference(pipeline, metricsService, r.Scheme); err != nil {
+		return err
+	}
+	if err = utils.CreateOrUpdateService(ctx, r.Client, metricsService); err != nil {
+		return fmt.Errorf("failed to create otel collector metrics service: %w", err)
```

Comment:
I think `Otel Collector` should be capitalized, if it's the one from https://opentelemetry.io/docs/collector/